Forwarded: https://github.com/fossfreedom/alternative-toolbar/commit/50cbad34b514551cf494bad787b5652f68586c93
Author: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: Wed, 26 Aug 2020 20:01:01 +0100
Description: [PATCH] No longer explicitly hide the Import button (Closes: #158)
 v3.4.4 changes mean we shouldn't explicitly hide the Import button that was
 previously visible by other means.

---
 alttoolbar_controller.py | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/alttoolbar_controller.py b/alttoolbar_controller.py
index 56f8f0f..26c5e54 100644
--- a/alttoolbar_controller.py
+++ b/alttoolbar_controller.py
@@ -317,15 +317,6 @@ class AltMusicLibraryController(AltGenericController):
 
         return "LibrarySource" in type(source).__name__
 
-    def hide_controls(self, source):
-        super(AltMusicLibraryController, self).hide_controls(source)
-
-        val, import_button = self.header.has_button_with_label(source,
-                                                               _('Import'))
-
-        if val:
-            import_button.set_visible(False)
-
 
 class AltSoundCloudController(AltGenericController):
     """
-- 
2.25.1

